tests/README.md

```diff
@@ -10,9 +10,12 @@ cp tests/ci/.env.default .env
 cd tests 
 docker-compose up -d
 cd ..
+php tests/_ci/generage-db-schemas.php
```

Comment:
Can you codecept build here as well.
And maybe we should prefix it with vendor/bin/codecept